### ISSUE TITLE
Idea: Encryption Features

### ISSUE THREAD

**Issue by supermacro** [parlez-vous/roadmap#7]
I've been thinking more and more about why data couldn't be encrypted at rest. 

And in fact .. I wonder why this isn't just the norm? I'm inspired by [Wormhole](https://wormhole.app) in this regard.

And I think it could be a selling point as well.

**Comment by supermacro** [comment]
What do you think @aszenz

**Comment by aszenz** [comment]
I'm not quite sure what e2e encryption means in this context, comments on blog posts are shown in public and are visible openly, as a site owner if the comments made on my site are leaked what problems would that cause?

Also there's a technical part regarding storing secrets, Wormhole for example embeds the secret key in the link for the file so that anyone with the link can decrypt the contents. 
Whatsapp stores the secret key on the mobile device itself and messages are decrypted there after arriving.
Where would parlez-vous store the keys

**Comment by supermacro** [comment]
Yeah this idea is totally not fleshed out and needs more thought. 

More than anything, I want to spark a discussion around privacy and protecting user data. Not entirely sure what that looks like in the context of commenting systems. 

Perhaps if there is a feature to allow private commenting then encryption might be of more relevance.
